firmware/Reload Pro.cydsn/ui.c

```diff
@@ -108,13 +112,24 @@ const menudata choose_readout_menu = {
 	}
 };
 
+const menudata tools_menu_data = {
+	NULL,
+	{
+		{"Batt ISR.", {NULL, (void*)1, 0}},		

```

Comment:
See my comment above; this should be implemented as a new state with a state func for the battery ISR entry.